tinychat/README.md

```diff
@@ -143,10 +141,28 @@ We recently evaluated AWQ's performance on the Visual Language Models. Here is a
 | FP16      | 84.3  | 64.6 | 62.2   | 87.2  | 73.6  | 87.3 | 1726.82 | 82.4 | 80.2   | 69.1 |
 | AWQ-INT4   | 84.1  | 64.4 | 61.3   | 86.7  | 73.2  | 88.2 | 1714.79 | 83.2 | 79.6   | 68.9 | 
 
+## New Optimization of Context Stage
+We have optimized the speed of the context stage and updated our code with several enhancements, including the adoption of flash attention and the elimination of redundant computations. The key optimizations include:
+1. Adopting the flash-attention kernel (currently supports only single-batch operations).
+2. Computing only the last tokens in the final logits layer.
+3. Utilizing history KV caches in the context stage to speed up. (chunk prefill)
+
+Under certain conditions, these improvements can result in up to an 8x speedup in TTFT(Time To the First Token) with minor accuracy loss comparing with the old version of TinyChat. We conducted experiments on the Orin GPU, and the detailed results are provided below. 
+
+![](./figures/TTFT_Speedup_flash_only_last_logits.png)
```

Comment:
Could we have a table instead of figures here? Let me know what's your opinion. @kentang-mit